Classes/Service/IndexingService.php

```diff
@@ -133,25 +137,21 @@ public function resetIndex(int $language = null): void
     {
         $this->assertConnectionHealthy();
 
-        $indexers = $this->indexerFactory->makeIndexers();
         $indexManager = IndexManager::getInstance();
 
         if ($language !== null) {
-            $indexManager->resetIndex(ExtconfService::getIndex($language));
+            $indeces = ExtconfService::getIndecesByLanguage($language);
```

Comment:
```suggestion
            $indices = ExtconfService::getIndecesByLanguage($language);
```

... and possibly other places.